### DISCUSSION TITLE
Artillery Shell containing with Fluid

### DISCUSSION THREAD

**Discussion by FOlkvangrField** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1315] [Q&A]
Inspired by the Disperser Canister, I plan to add an Artillery Shell containing fluids that will produce an AoE effect based on the fluid after hitting a target, similar to the Disperser Canister. Is this a good idea?

**Comment by 9Valjew** [comment]
https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/pull/1320

**Comment by 9Valjew** [comment]
(inspired by this)
